src/errors.js

```diff
@@ -0,0 +1,29 @@
+export class KisiError extends Error {
+    constructor(status, code = "000000", reason = null) {
+        super("An error occurred interacting with the Kisi API.")
+
+        this.status = status
+        this.code = code
+        this.reason = reason
+    }
+}
+
+export function handleError(error) {
```

Comment:
I decided to seperate them and moved the handler in a seperate file since it is not only used in index.js